test/parallel/test-process-exit-code-validation.js

```diff
@@ -0,0 +1,85 @@
+'use strict';
+
+require('../common');
+
+const args = [
+  // invaild code
+  {
+    code: '',
+    expected: 1,
+  },
+  {
+    code: '1 one',
+    expected: 1,
+  },
+  {
+    code: 'two',
+    expected: 1,
+  },
+  {
+    code: {},
+    expected: 1,
+  },
+  {
+    code: [],
+    expected: 1,
+  },
+  {
+    code: true,
+    expected: 1,
+  },
+  {
+    code: false,
+    expected: 1,
+  },
+  // vaild code
+  {
+    code: undefined,
+    expected: 0,
+  },
+  {
+    code: null,
+    expected: 0,
+  },
+  {
+    code: 0,
+    expected: 0,
+  },
+  {
+    code: 1,
+    expected: 1,
+  },
```

Comment:
Can we add a test with a non-integer value?
```suggestion
  {
    code: 2.1,
    expected: 1,
  },
  {
    code: Infinity,
    expected: 1,
  },
  {
    code: NaN,
    expected: 1,
  },
```

Reply:
Updated.